### PR TITLE
update readme for rtags with helm

### PR DESCRIPTION
**Pull request by tddsg**
Hi,

I think that the readme for `rtags` with `helm` is kind of outdated.
I updated it with the code that is working for me at the moment.

Can you check if it is helpful?
Thanks!

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -995,8 +995,8 @@ To turn off the automatic Flycheck syntax checking, set the variable
   To integrate rtags with helm you need to do the following:
 
 #+BEGIN_SRC emacs-lisp
-(require 'rtags-helm)
-(setq rtags-use-helm t)
+(require 'helm-rtags)
+(setq rtags-display-result-backend 'helm)
 #+END_SRC
```
Comment (dubnde) [inline README.org]: Consider add similar for `ivy` using `ivy-rtags`

Comment (tddsg) [inline README.org]: I see that `ivy` is supported by `rtags`, but not sure why the authors haven't provided a readme for it.

Comment (cslux) [inline README.org]: Will add something. @tddsg  Someone else has contributed ivy to rtags

**Comment by cslux** [comment]
Thanks, merged!

**Pull request merged**